lib/rex/proto/apache_j_p.rb

```diff
@@ -0,0 +1,128 @@
+# -*- coding: binary -*-
+
+require 'bindata'
+
+# @see: https://tomcat.apache.org/tomcat-3.3-doc/ApacheJP.html
```

Comment:
Super weird, that was live within the past 24 hours. I can still remember what it looks like because I read through it a few times while working on this. I changed it to a different link though that seems to have the same information.